components/textfield/index.css

```
@@ -131,207 +460,233 @@ governing permissions and limitations under the License.
-    inset-block-end: calc(
-      calc(var(--spectrum-textfield-texticon-height) / 2) -
-        calc(var(--spectrum-textfield-texticon-invalid-icon-height) / 2)
-    );
+    /* Treat all quiet inputs and text areas the same */
+    resize: none;
+    overflow-y: hidden;
+   }
 
-    inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, var(--spectrum-textfield-texticon-invalid-icon-margin-left));
+  /****** Input - Quiet 🤫 + Disabled 🚫 ******/
+  .spectrum-Textfield--quiet.is-disabled &,
+  .spectrum-Textfield--quiet.is-disabled:hover &,
+  &:disabled {
+    background-color: transparent;
+    border-color: var(--mod-textfield-border-color-disabled, var(--spectrum-textfield-border-color-disabled));
+    color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-textfield-text-color-disabled)));
 
-    .spectrum-Textfield--quiet& {
-      inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, 0);
+    &::placeholder {
+      color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-textfield-text-color-disabled)))
     }
   }
 
-  .spectrum-Textfield.is-valid & {
-    inline-size: var(--spectrum-textfield-texticon-success-icon-width);
-    block-size: var(--spectrum-textfield-texticon-success-icon-height);
-    inset-block-end: calc(
-      calc(var(--spectrum-textfield-texticon-height) / 2) -
-        calc(var(--spectrum-textfield-texticon-success-icon-height) / 2)
-    );
-    inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, var(--spectrum-textfield-texticon-success-icon-margin-left));
+  /****** Input ReadOnly ******/
+  .spectrum-Textfield.is-readOnly &,
+  .spectrum-Textfield.is-readOnly:hover &,
+  &:read-only {
+    background-color: transparent;
+    border-color: transparent;
+    color: var(--highcontrast-textfield-text-color-readonly, var(--mod-textfield-text-color-readonly, var(--spectrum-textfield-text-color-readonly)));
 
-    .spectrum-Textfield--quiet& {
-      inset-inline-end: var(--spectrum-textfield-icon-inline-end-override, 0);
+    &::placeholder {
+      color: var(--highcontrast-textfield-text-color-readonly, var(--mod-textfield-text-color-readonly, var(--spectrum-textfield-text-color-readonly)));
+      background-color: transparent;
     }
   }
 }
 
-/* styles the left icon for textfield, assumes usage of workflow icon sizing (18px by 18px) */
-.spectrum-Textfield-icon {
-  display: block;
-  position: absolute;
-  block-size: var(--spectrum-alias-workflow-icon-size-m);
-  inline-size: var(--spectrum-alias-workflow-icon-size-m);
-  inset-inline-start: var(--spectrum-textfield-texticon-padding-left);
-  inset-block-start: calc(
-    var(--spectrum-textfield-texticon-height) / 2 -
-      var(--spectrum-alias-workflow-icon-size-m) / 2
-  );
+/*** Layout Variant - Side Label ***/
+.spectrum-Textfield--sidelabel {
+  .spectrum-Textfield-input-wrapper {
```

```suggestion
.spectrum-Textfield--sideLabel {
  .spectrum-Textfield-inputWrapper {
```

^ I think the naming here needs to be updated